src/utils/sumo_changeset.erl

```diff
@@ -0,0 +1,464 @@
+%%% @doc Based on Elixir `Ecto.Changeset'.
```

Comment:
This module has too many functions (28). Consider breaking it into a number of modules.